src/components/ButtonAppBar/index.js

```diff
@@ -42,29 +42,35 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
-const divStyle = {
-  display: 'inline',
-}
+const divStyle = { display: 'inline' }
 
 export function ButtonAppBar() {
+  const classes = useStyles()
+  const history = useHistory()
+  const pageHistory = useHistory()
   const { userInfo, logout } = useUserContext()
 
+  const handleTitle = event => {
+    pageHistory.push('/ItemList')
```

Comment:
This is redundant with the next line.
We don't want to push the same path twice here.